### PR TITLE
Java nd

### PR DESCRIPTION
**Pull request by enn-dee**
Hello 
I have added some basic content to the project . The content covers the following topics:
- Introduction About JAVA
- Basic Syntax 
- Basic Syntax /1. Comments . /2 Variables . /3 Data Types.
-Please let me know if you have any feedback or suggestions for improvement.

### CODE REVIEW

File: JAVA/1.Intro/intro.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Java is a programming language and computing platform first released by
```
Comment (ShehrozIrfan) [inline JAVA/1.Intro/intro.md:1]: Good work.

File: JAVA/2.Basic Syntax/Basic Syntax.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+
+# Java Basic Syntax
+Here are some basic syntax elements of the Java programming language:
+
+
```
Comment (ShehrozIrfan) [inline JAVA/2.Basic Syntax/Basic Syntax.md]: Remove the extra lines from the whole file. There must be only one line space if needed.

File: JAVA/2.Basic Syntax/Basic Syntax.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ Java has several built-in data types, including integers (int), 
+ floating-point numbers (float and double), characters (char), 
+ and boolean values (boolean). For example:
+
+ ```java
+int x = 10;          // Integer
+float pi = 3.14;     // Floating-point number
+char letter = 'A';   // Character
+boolean flag = true; // Boolean value
+
```
Comment (ShehrozIrfan) [inline JAVA/2.Basic Syntax/Basic Syntax.md]: Remove this extra line.

File: JAVA/2.Basic Syntax/Basic Syntax.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ Variables are used to store values in a program. 
+ In Java, you must declare a variable before you can use
+  it and specify its data type. **For example:**
+  ```java
+int x;         // Declare an integer variable named x
+x = 10;        // Assign the value 10 to x
+
+String name;   // Declare a string variable named name
+name = "John"; // Assign the value "John" to name
+
```
Comment (ShehrozIrfan) [inline JAVA/2.Basic Syntax/Basic Syntax.md]: Remove line

File: JAVA/2.Basic Syntax/Basic Syntax.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+extend to the end of the line. Multi-line comments start with /* 
+and end with */ .
+**Example**  
+// This is a single-line comment
+
+/*
+This is a
+multi-line comment
+*/
+# Variables
```
Comment (ShehrozIrfan) [inline JAVA/2.Basic Syntax/Basic Syntax.md]: Add a line break above this line, and heading should be: `## 3. Variables`

**Review by ShehrozIrfan — CHANGES_REQUESTED** [review]
Overall looks good. Good work.

Just need some changes:
- Remove the extra line breaks
- Add the proper formatting so that it looks good

**Comment by enn-dee** [comment]
Review Changes

**Review by ShehrozIrfan — APPROVED** [review]
Looks good to merge 🎉
Thank you for your contribution 🚀

**Pull request merged**